chapter04/Task_4_3.py

```diff
@@ -0,0 +1,67 @@
+# Word Jumble
+#
+# The computer picks a random word and then "jumbles" it
+# The player has to guess the original word
+
+import random
+
+# create a sequence of words to choose from
+WORDS = ("python", "jumble", "easy", "difficult", "answer", "xylophone")
+# pick one word randomly from the sequence
+word = random.choice(WORDS)
+# create a variable to use later to see if the guess is correct
+correct = word
+# create tips
+TIPS = ("The language we learn to program in",
+        "What did we do with the letters of the words?!",
+        "What do we say when we win CS: GO?",
+        "The kid who got caught stealing - ",
+        "What do we want to get to the question - ",
+        "Strange musical instrument")
+# create score
+score = 100
+
+# create a jumbled version of the word
+jumble = ""
+while word:
```

Comment:
That solution is good, but "random" module has built-in shuffling function.
It works with lists (not strings) so unfortunately you'll have to adapt.
This how this could be done:
>>> word = "abcdef"
>>> word_characters = list(word)
>>> random.shuffle(word_characters)
>>> "".join(word_characters)

Please learn how this works.
The result code is not much shorter than you have, but should be more efficient (try with text of 10000 symbols).